pmd-java/src/test/java/net/sourceforge/pmd/lang/java/multifile/JavaMultifileVisitorTest.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.multifile;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Test;
+
+import net.sourceforge.pmd.lang.LanguageVersionHandler;
+import net.sourceforge.pmd.lang.java.ParserTstUtil;
+import net.sourceforge.pmd.lang.java.ast.ASTCompilationUnit;
+import net.sourceforge.pmd.lang.java.ast.ASTMethodDeclaration;
+import net.sourceforge.pmd.lang.java.ast.JavaParserVisitorAdapter;
+import net.sourceforge.pmd.lang.java.ast.JavaQualifiedName;
+import net.sourceforge.pmd.lang.java.multifile.signature.JavaFieldSigMask;
+import net.sourceforge.pmd.lang.java.multifile.signature.JavaOperationSigMask;
+import net.sourceforge.pmd.lang.java.multifile.signature.JavaSignature.Visibility;
+import net.sourceforge.pmd.lang.java.multifile.testdata.MultifileVisitorTestData;
+
+/**
+ * Tests of the metrics visitor.
+ *
+ * @author Clément Fournier
+ */
+public class JavaMultifileVisitorTest {
+
+
+    @Test
+    public void testPackageStatsNotNull() {
+        assertNotNull(PackageStats.INSTANCE);
+    }
+
+
+    @After
```

Comment:
It might be safer to use `@Before` here - I assume, you wrote the tests expecting that the ProjectMirror is empty. If there are other tests, which forgot to cleanup after themselves, it would fail.